samtranslator/model/api/api_generator.py

```diff
@@ -280,7 +282,7 @@ def _construct_rest_api(self):
             rest_api.BodyS3Location = self._construct_body_s3_dict()
         elif self.definition_body:
             # # Post Process OpenApi Auth Settings
-            self.definition_body = self._openapi_postprocess(self.definition_body)
+            self.definition_body = self._openapi_postprocess(self.swagger_editor.swagger)
```

Comment:
This is now the only time we update self.definition_body with the result of all of the changes we made with the SwaggerEditor